src/shared/rating/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { Option } from 'src/shared/forminator/types';
```

Comment:
It's better to be on src/shared/utils/getOptionsFromDictionary.ts and have it imported from it